.github/workflows/manylinux.yml

```diff
@@ -11,41 +11,46 @@ jobs:
     strategy:
       matrix:
         python_path: [ cp37-cp37m, cp38-cp38, cp39-cp39, cp310-cp310 ]
+        arch_image: [ keyvidev/manylinux-builder-x86_64, keyvidev/manylinux-builder-aarch64 ]
 
     runs-on: ubuntu-latest
-    container:
```

Comment:
Why was this deleted ? 

If we delete this, then I'm afraid the `Publish package` would break as that assumes use of specific python installation and path, that is available inside the manylinux docker container, and not on the host.

Reply:
I'm afraid I couldn't use top-level ARM container here, that's why I use a custom action